zeebe/dynamic-config/src/main/java/io/camunda/zeebe/dynamic/config/changes/PartitionDeleteExporterApplier.java

```diff
@@ -96,16 +97,25 @@ public Either<Exception, UnaryOperator<MemberState>> initMemberState(
 
   @Override
   public ActorFuture<UnaryOperator<MemberState>> applyOperation() {
-    return partitionChangeExecutor
+    final var result = new CompletableActorFuture<UnaryOperator<MemberState>>();
+    partitionChangeExecutor
         .deleteExporter(partitionId, exporterId)
-        .thenApply(
-            (nothing) -> {
-              return memberState ->
-                  memberState.updatePartition(
-                      partitionId,
-                      partition ->
-                          partition.updateConfig(config -> deleteExporter(config, exporterId)));
+        .onComplete(
```

Comment:
❓ Why the change to this method?

Reply:
The method in the original PR is not available stable/8.7 branch.